modules/sonar/Makefile

```diff
@@ -24,15 +24,20 @@ sonar/go/prow: export SONAR_TOKEN="$(shell cat /etc/sonarcloud/token 2> /dev/nul
 sonar/go/prow:
 	@echo "Verify sonar token from kube secret has been mounted" | tee -a "${ARTIFACT_DIR}/sonar.log"
 	@if [[ -z "${SONAR_TOKEN}" ]] ; then echo "---> ERROR: SONAR_TOKEN is not set" | tee -a "${ARTIFACT_DIR}/sonar.log" ; exit 1 ; fi
-	
-	@echo "Running go test" | tee -a "${ARTIFACT_DIR}/sonar.log"
-	go test -coverprofile=coverage.out -json ${SONAR_GO_TEST_ARGS} | tee -a report.json "${ARTIFACT_DIR}/sonar_gotest.json"
+
+	@if [[ -z "${SONAR_GO_TEST_USE_GINKGO}" ]]; then \
+		echo "Running go tests" | tee -a "${ARTIFACT_DIR}/sonar.log"; \
+		go test -coverprofile=coverage.out -json ${SONAR_GO_TEST_ARGS} | tee -a report.json "${ARTIFACT_DIR}/sonar_gotest.json"; \
+	else \
+		echo "Running ginkgo tests" | tee -a "${ARTIFACT_DIR}/sonar.log"; \
+		ginkgo -r --cover --coverprofile=coverage.out --coverpkg ${SONAR_GO_TEST_ARGS} | tee -a report.json "${ARTIFACT_DIR}/sonar_gotest.json"; \
```

Comment:
This is being tee-ed to a `json` file, but I don't see a JSON flag here. Does Ginko default to JSON reports? Or should this remove the pipe to `tee` and use `--json-report=report.json` and then optionally do `cp report.json ${ARTIFACT_DIR}/sonar_gotest.json` (since I don't think Gingo is putting JSON on the screen)?

Reply:
Good catch!  A little too much cut and paste.  I made a change to save a coverage.html like we do when run locally